include/sys/util.h

```diff
@@ -466,6 +466,11 @@ uint8_t u8_to_dec(char *buf, uint8_t buflen, uint8_t value);
  * to replacement lists being empty or containing numbers or macro name
  * like tokens.
```

Comment:
A problem is that `IS_EMPTY(__VA_ARGS__)` doesn't work either unless there are zero or one arguments.  The above should also be reworded: it's not checking whether a macro that evaluates to a replacement list is empty, it's checking whether a macro evaluates to limited type of non-empty token sequence.

Maybe:
>  Note that this macro is intended to be used when iterating over macro arguments to determine whether a single argument evalutes to (true:) an empty token sequence or (false:) a sequence that begins with a number or identifier and does not include a comma.  Arguments that evaluate to anything else may produce a build error.